### PR TITLE
updated speech settings panel and adding default values

### PR DESCRIPTION
**Pull request by AKissmehl**
This PR includes the changes from [PR #2063](https://github.com/WEKIT-ECS/MIRAGE-XR/pull/2063), ensuring that the improvements and fixes from that PR are incorporated. In addition to those changes, this PR introduces a rework of the remaining UI components, including the sub-pages. The UI elements have been refactored into prefabs. Default values have also been set for the menu to enhance usability. Moreover, this PR removes the reliance on built-in functions, replacing them with alternative solutions.

Key Changes:

-     Merged changes from [PR #2063](https://github.com/WEKIT-ECS/MIRAGE-XR/pull/2063).
-     Refactored UI components into prefabs.
-     Set default values for UI prefabs to standardize usage.
-     Removed the use of built-in functions, opting for custom solutions.

### CODE REVIEW

**Comment by Alvareon** [comment]
missing prefab of header:
Screenshot 2024-08-22 at 21 18 06

**Comment by AKissmehl** [comment]
@fominykh I ran in to a UX Issue. How to Handel that situation, would the header scroll away as well, or should it be covered? 

https://github.com/user-attachments/assets/b4076b28-d7a7-41d8-9c72-127ba6a9f059

File: Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/Mobile/SpeechSettings.cs [hunk 1]
```diff
@@ -49,6 +51,97 @@ public class SpeechSettings : MonoBehaviour
         /// </summary>
         private AIModel _languageData;
 
+        /// <summary>
+        /// Starts the SpeechSettings by setting default values.
+        /// </summary>
+
+        [SerializeField] 
+        private GameObject PromtView;
```
Comment (fwild) [inline Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/Mobile/SpeechSettings.cs]: PromptView

File: Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/Mobile/SpeechSettings.cs [hunk 2]
```diff
@@ -49,6 +51,97 @@ public class SpeechSettings : MonoBehaviour
...
+        private GameObject LLMView;
+
+        [SerializeField]
+        private GameObject STTView;
+
+        [SerializeField]
+        private GameObject TTSView; 
+        
+        [SerializeField]
+        private Button  PromtViewBackBtn;
```
Comment (fwild) [inline Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/Mobile/SpeechSettings.cs]: PromptViewBackBtn

**Comment by Alvareon** [comment]
@AKissMail pls add meta file for the Header.prefab
Screenshot 2024-08-23 at 16 19 52

**Review by fwild — APPROVED** [review]

**Comment by Alvareon** [comment]
@AKissMail pls add another meta file:
Screenshot 2024-08-29 at 13 09 32

**Review by Alvareon — APPROVED** [review]

**Pull request merged**